x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/profiler/ilm-policy/profiling.json

```diff
@@ -19,8 +17,8 @@
         "set_priority": {
           "priority": 50
         },
-        "shrink": {
```

Comment:
Shrinking number of shards on the warm tier would reduce the total number of shards a bit but as Elasticsearch got much better handling larger shard counts (see https://www.elastic.co/blog/benchmark-driven-optimizations-scalability-elasticsearch-8 for more background) we avoid that step. Instead, force merging will help reducing storage size and that's why we added it here.